man/less-variables.xml

```diff
@@ -94,10 +94,12 @@
     <varlistentry id='colors'>
       <term><varname>$SYSTEMD_COLORS</varname></term>
 
-      <listitem><para>The value must be a boolean. Controls whether colorized output should be
-      generated. This can be specified to override the decision that <command>systemd</command> makes based
-      on <varname>$TERM</varname> and what the console is connected to.</para>
-      </listitem>
+      <listitem><para>Takes a boolear argument. When true, <command>systemd</command> and related utilities
+      will use colors in their output, otherwise the output will be monochrome. Additionally, the variable can
+      take one of the following special values: <literal>16</literal>,<literal>256</literal> to restrict the use
+      of colors to the base 16 or 256 ANSI colors, respectively. This can be specified to override the decision
+      that <command>systemd</command> makes based on <varname>$TERM</varname> and what the console is connected
```

Comment:
This applies to all executables, not just `systemd`. So please say "`This can be specified to override the automatic decision based on <varname>$TERM</varname> and what the console is connected to.`"

Reply:
Done.